src/psyclone/transformations.py

```diff
@@ -154,20 +156,15 @@ def validate(kern, options=None):
                     "kernel.".format(kern.name, var.name)), err)
 
 
-class LoopFuseTrans(Transformation):
+class LoopFuseTrans(LoopTrans):
```

Comment:
Yes, I wasn't too sure about this either but, as you say, it seems to be quite a nice solution and I can't think of any drawbacks.

Reply:
Is this more of a "has a" rather than an "is a". i.e. would it be possible for LoopFuseTrans to be a Transformation as it was before and contain an instance of LoopTrans which it could use to perform performance validation on individual nodes?

Reply:
Coming back to this and re-reading the descriptions of the two transformations I feel happier now about having `LoopFuseTrans` as a sub-class of `LoopTrans` - it does act on loops and returns a loop. Added to which, currently `LoopTrans` is abstract so we can't do "have a" without making it concrete (is that the correct opposite of 'abstract'?). I'm happy to put this out for further comment from @hiker and @sergisiso.

Reply:
I've discussed this with @sergisiso and the consensus is that your approach is the only feasible way given the way that transformation are implemented - and of course it is quite elegant. Therefore I am accepting it.